.goreleaser.yml

```diff
@@ -4,11 +4,11 @@ blobs:
     folder: "releases/{{.Version}}"
 
 builds:
-  - main: ./cmd/calyptia
+  - main: .
     id: calyptia
     binary: calyptia
     ldflags:
-      - -s -w -X main.version={{.Version}}
+      - -s -w -X github.com/calyptia/cli/cmd/version.Version={{.Version}}
```

Comment:
@patrick-stephens is this value correct? I thought it was supposed to be `{{.Env.Version}}`